gearcmd/worker.go

```diff
@@ -114,6 +114,9 @@ func (conf TaskConfig) doProcess(job baseworker.Job) error {
 		// Will be nil if the channel was closed without any errors
 		return err
 	case <-time.After(conf.CmdTimeout):
+		if err := cmd.Process.Kill(); err != nil {
+			return fmt.Errorf("error killing timed out process: %s", err.Error())
+		}
```

Comment:
message is a little ambiguous/coarse . Maybe `"Process timeout after %d. Error: %s", conf.CmdTimeout, err.Error()`  ?